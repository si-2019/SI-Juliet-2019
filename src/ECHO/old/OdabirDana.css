.OdabirDana {
    text-align: center;
  }
  .App-header {
    background-color: #282c34;
    min-height: 15vh;
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    font-size: 10px;
    color: white;
  }
  #generisi {
    border-radius: 8px;
    background-color:  #282c34;
    font-size: 9px;
    color: white;
    width : 15vw;
    float : right;
    margin-right : 5vw;
    margin-top : 5vw;
    border: 1px solid black; 
  }
  #pon, #uto, #sri, #cet, #pet{
    border-radius: 8px;
    background-color:  #282c34;
    float : left;
    min-height: 15vh;
    margin-left: 4vw;
    margin-top: 25vh;
    width : 15vw;
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    font-size: 10px;
    color: white;
    border : 1px solid, black;
  }
  .App-edit {
    border: 1px solid black; 
    border-radius: 4px;
    background-color: darkcyan;
    min-height: 2vh;
    float: left;
    width: 10vw;
    flex-direction: column;
    align-items: center;
    margin-top: 5vh;
    justify-content: center;
    font-size: 7px;
    color: white;
  }
  .App-ponisti {
    border: 1px solid black; 
    border-radius: 4px;
    background-color: grey;
    min-height: 2vh;
    float: left;
    width: 10vw;
    flex-direction: column;
    align-items: center;
    margin-top: 1vh;
    justify-content: center;
    font-size: 7px;
    color: white;
  }
  #dugmad{
    width: 100%;
  }
  #polovina{
    width:20%;
    float:left;
  }
  #polovina2{
    width:65%;
    float:right;
   
  }
  #id1{
    width:82%;
    float:left;
    clear:left;
  }
  #id2{
    width:18%;
    float:right;
    clear:right;
  }
  #cetvrtina{
    width:60%;
    float:left;
  }
  #cetvrtina2{
    width:40%;
    float:right;
  }
  #id3{
    width:50%;
    float:left;
    clear:left;
  }
  #id4{
    width:50%;
    float:right;
    clear:right;
  }
  #gen{
    clear:both;
  }
  #b1{
    margin-left: 6.5vw;
  }
  #b2, #b3, #b4, #b5{
    margin-left: 9vw;
  }
  #b6{
    margin-left: 6.5vw;
  }
  #b7, #b8, #b9, #b10{
    margin-left: 9vw;
  }