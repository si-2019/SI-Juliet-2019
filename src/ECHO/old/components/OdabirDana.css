.App {
  text-align: center;
}
.App-header {
  background-color: #282c34;
  min-height: 15vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: 10px;
  color: white;
}
#generisi {
  border-radius: 8px;
  background-color:  #282c34;
  font-size: 9px;
  color: white;
  width : 15vw;
  float : right;
  margin-right : 5vw;
  margin-top : 3vw;
  border: 1px solid black; 
}
#pon, #uto, #sri, #cet, #pet{
  border-radius: 8px;
  background-color:  #282c34;
  float : left;
  min-height: 15vh;
  margin-left: 4vw;
  margin-top: 25vh;
  width : 15vw;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: 10px;
  color: white;
  border : 1px solid, black;
}
.App-edit {
  border: 1px solid black; 
  border-radius: 4px;
  background-color: darkcyan;
  min-height: 2vh;
  float: left;
  width: 10vw;
  flex-direction: column;
  align-items: center;
  margin-top: 5vh;
  justify-content: center;
  font-size: 7px;
  color: white;
}
.App-ponisti {
  border: 1px solid black; 
  border-radius: 4px;
  background-color: grey;
  min-height: 2vh;
  float: left;
  width: 10vw;
  flex-direction: column;
  align-items: center;
  margin-top: 1vh;
  justify-content: center;
  font-size: 7px;
  color: white;
}
#b1{
  margin-left: 6.5vw;
}
#b2, #b3, #b4, #b5{
  margin-left: 9vw;
}
#b6{
  margin-left: 6.5vw;
}
#b7, #b8, #b9, #b10{
  margin-left: 9vw;
}