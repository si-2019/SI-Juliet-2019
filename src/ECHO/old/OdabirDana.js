import React, { Component } from 'react';
import { Alert } from "reactstrap";
import './OdabirDana.css';
 
var siva = '#282c34';
var zelena= '#30e20c';
class OdabirDana extends Component {
  constructor(){
    super()
    this.state={
      prikaziMe: false,
      prikaziMe2: false,
      prikaziMe3: false,
      prikaziMe4: false,
      prikaziMe5: false,
      color: siva,
      color2:siva,
      color3: siva,
      color4: siva,
      color5: siva
    }
  }
 
  operation(){
    this.setState({
      prikaziMe: true,
      color: zelena
    })
  }
  operation2(){
    this.setState({
      prikaziMe2:true,
      color2:zelena
    })
  }
  operation3(){
    this.setState({
      prikaziMe3:true,
      color3:zelena
    })
  }
  operation4(){
    this.setState({
      prikaziMe4:true,
      color4:zelena
    })
  }
  operation5(){
    this.setState({
      prikaziMe5:true,
      color5: zelena
    })
  }
  ponisti1(){
    alert('Uspjesno ste ponistili termine za ponedjeljak')
    this.setState({
      prikaziMe:false,
      color: siva,
    })
  }
  ponisti2(){
    alert('Uspjesno ste ponistili termine za utorak')
    this.setState({
      prikaziMe2:false,
      color2: siva,
    })
  }
  ponisti3(){
    alert('Uspjesno ste ponistili termine za srijedu')
    this.setState({
      prikaziMe3:false,
      color3: siva,
    })
  }
  ponisti4(){
    alert('Uspjesno ste ponistili termine za cetvrtak')
    this.setState({
      prikaziMe4: false,
      color4:siva,
    })
  }
  ponisti5(){
    alert('Uspjesno ste ponistili termine za petak')
    this.setState({
      prikaziMe5:false,
      color5:siva,
    })
  }
  unesi(){
    alert('Uspjesno ste spremili promjene')
  }
   render() {
     let boja=this.state.color ;
     let boja2=this.state.color2;
     let boja3=this.state.color3;
     let boja4=this.state.color4;
     let boja5=this.state.color5;
    return (
      <div className="App">
        <header className="App-header">
          <h1> Klikom na odgovarajući pravougaonik odaberite dane u kojima imate slobodne termine.</h1>
        </header>
        <div id = "pon" onClick={()=>this.operation()} style={{backgroundColor: boja}}><h1>PONEDJELJAK</h1></div>
        <div id = "uto" onClick={()=>this.operation2()} style={{backgroundColor: boja2}}><h1>UTORAK</h1></div>
        <div id = "sri" onClick={()=>this.operation3()} style={{backgroundColor: boja3}}><h1>SRIJEDA</h1></div>
        <div id = "cet" onClick={()=>this.operation4()} style={{backgroundColor: boja4}}><h1>ČETVRTAK</h1></div>
        <div id = "pet" onClick={()=>this.operation5()} style={{backgroundColor: boja5}}><h1>PETAK</h1></div>
        <div id="dugmad">
        <div id="polovina">
        {this.state.prikaziMe?
        <div id="id1">
        <button className="App-edit" id = "b1"  ><h2>Unesite termin/e</h2></button>
        <button className="App-ponisti" id = "b6" onClick={()=>this.ponisti1()}><h2>Poništite dan</h2></button>
        </div>
                :null
        }
         {this.state.prikaziMe2?
         <div id="id2">
        <button className="App-edit" id = "b2" ><h2>Unesite termin/e</h2></button>
        <button className="App-ponisti" id = "b7" onClick={()=>this.ponisti2()}><h2>Poništite dan</h2></button>
        </div>
        :null
        }
        </div>
        <div id="polovina2">
        <div id="cetvrtina">
        {this.state.prikaziMe3?
        <div id="id3">
        <button className="App-edit" id = "b3"><h2>Unesite termin/e</h2></button>
        <button className="App-ponisti" id = "b8" onClick={()=>this.ponisti3()}><h2>Poništite dan</h2></button>
        </div>
        :null
        }
        {this.state.prikaziMe4?
        <div id="id4">
        <button className="App-edit" id = "b4"><h2>Unesite termin/e</h2></button>
        <button className="App-ponisti" id = "b9" onClick={()=>this.ponisti4()}><h2>Poništite dan</h2></button>
        </div>
        :null
        }
        </div>
        <div id="cetvrtina2">
        {this.state.prikaziMe5?
        <div id="id5">
        <button className="App-edit" id = "b5"><h2>Unesite termin/e</h2></button>
        <button className="App-ponisti" id = "b10" onClick={()=>this.ponisti5()}><h2>Poništite dan</h2></button>
        </div>
        :null
        }
        </div>
        </div>
        </div>
        <div id="gen">
        <button id = "generisi" onClick={()=>this.unesi()}><h1>Generiši raspored</h1></button>
        </div>
      </div>
    );
  }
}
export default OdabirDana;