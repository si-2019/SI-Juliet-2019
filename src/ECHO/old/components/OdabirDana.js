import React, { Component } from 'react';
import './OdabirDana.css';
class OdabirDana extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <h1> Klikom na odgovarajući pravougaonik odaberite dane u kojima imate slobodne termine.</h1>
        </header>
        <div id = "pon"><h1>PONEDJELJAK</h1></div>
        <div id = "uto"><h1>UTORAK</h1></div>
        <div id = "sri"><h1>SRIJEDA</h1></div>
        <div id = "cet"><h1>ČETVRTAK</h1></div>
        <div id = "pet"><h1>PETAK</h1></div>
        <button className="App-edit" id = "b1"><h2>Unesite termin/e</h2></button>
        <button className="App-edit" id = "b2"><h2>Unesite termin/e</h2></button>
        <button className="App-edit" id = "b3"><h2>Unesite termin/e</h2></button>
        <button className="App-edit" id = "b4"><h2>Unesite termin/e</h2></button>
        <button className="App-edit" id = "b5"><h2>Unesite termin/e</h2></button>
        <button className="App-ponisti" id = "b6"><h2>Poništite dan</h2></button>
        <button className="App-ponisti" id = "b7"><h2>Poništite dan</h2></button>
        <button className="App-ponisti" id = "b8"><h2>Poništite dan</h2></button>
        <button className="App-ponisti" id = "b9"><h2>Poništite dan</h2></button>
        <button className="App-ponisti" id = "b10"><h2>Poništite dan</h2></button>
        <button id = "generisi"><h1>Generiši raspored</h1></button>
      </div>
    );
  }
}
export default OdabirDana;