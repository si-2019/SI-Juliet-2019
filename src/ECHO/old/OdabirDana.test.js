import React from 'react';
import OdabirDana from './OdabirDana';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import { shallow } from 'enzyme';
 
Enzyme.configure({ adapter: new Adapter() });


it("prikazuje formu za odabir dana", () => {
    const component = shallow(<OdabirDana />);
    expect(component).toMatchSnapshot();
  });
it("postoji li button za poniÅ¡tavanje dana", () => {
    const wrapper = shallow(<OdabirDana />);
    expect(wrapper.find("#b6").exists());
  });
it("postoji li button za odabir termina", () => {
    const wrapper = shallow(<OdabirDana />);
    expect(wrapper.find("#b1").exists());
  });
it("postoji li button za generisanje rasporeda", () => {
    const wrapper = shallow(<OdabirDana />);
    expect(wrapper.find("#generisi").exists()).toBe(true);
  });