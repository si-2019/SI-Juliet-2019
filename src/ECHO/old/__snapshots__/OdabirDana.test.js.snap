// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prikazuje formu za odabir dana 1`] = `ShallowWrapper {}`;
